yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -186,19 +186,23 @@ func checkSourceExporter(exporterRole string) bool {
 	return exporterRole == "source_db_exporter"
 }
 
-func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*string, exportSourceType string, formatIfRequired bool) error {
+func (conv *DebeziumValueConverter) convertMap(eventSchema string, tableName string, m map[string]*string, exportSourceType string, formatIfRequired bool) error {
 	var schemaRegistry *SchemaRegistry
+	targetTableName := tableName
 	if checkSourceExporter(exportSourceType) {
 		schemaRegistry = conv.schemaRegistrySource
 	} else {
+		if eventSchema != "public" {
+			targetTableName = fmt.Sprintf("%s.%s", eventSchema, tableName)
+		}
```

Comment:
Doesn't these kind of maybeQualifiedTableName requires to check the sourceDBType first and based on that...?